specificatie/beslagen.yaml

```diff
@@ -145,6 +145,9 @@ components:
           items:
             $ref: "domain.yaml#/components/schemas/PersoonBeperkt"
             description: "Partij die beslag legt op een kadastraal onroerende zaak, waardoor de zakelijk gerechtigde niets met de kadastraal onroerende zaak kan doen."
+        bedragVordering:
+          $ref: "domain.yaml#/components/schemas/Bedrag"
+          description: "Het bedrag waarvoor beslag is gelegd. Indien bij een beslag verschillende bedragen voorkomen, dan worden deze bedragen getotaliseerd."
```

Comment:
graag veranderen in:
Het bedrag waarvoor beslag is gelegd. Als een beslag bestaat uit verschillende bedragen, dan moet je die bij elkaar optellen. 

@HildeVos, klopt dat?

Reply:
> dan moet je die bij elkaar optellen

Dit is een description gericht op de gebruiker. Die krijgt altijd maar één bedrag, waarin de eventuele verschillende bedragen als het goed is al zijn opgeteld. Dan klopt deze voorgestelde description toch niet?

Reply:
ok, dan wordt het: 
Het bedrag waarvoor beslag is gelegd. Als een beslag bestaat uit verschillende bedragen, dan zijn die bij elkaar opgeteld.

@HildeVos, klopt dat?